### PR TITLE
qa analysis macros splitting

### PR DESCRIPTION
**Pull request by mmazzilli**

### CODE REVIEW

File: FirstAnalysis/HFmcvalidation.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/ usr / bin / env python
```
Comment (vkucera) [inline FirstAnalysis/HFmcvalidation.py:1]: This doesn't look right.

**Review by vkucera — CHANGES_REQUESTED** [review]

File: FirstAnalysis/HFmcvalidation.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+from ROOT import (  # pylint: disable=import-error, no-name-in-module, unused-import; TEfficiency,; TLegend,; gPad,
```
Comment (vkucera) [inline FirstAnalysis/HFmcvalidation.py:2]: Why are these checks disabled?

**Review by vkucera — CHANGES_REQUESTED** [review]

File: FirstAnalysis/trackingstudies.py [hunk 3]
```diff
@@ -136,42 +78,38 @@ def tracking_resolution(var1, mode, var2, var3):
...
+
+multiplicity()
+specie_distribution("pion", "pt")
+specie_distribution("proton", "pt")
+specie_distribution("electron", "pt")
+specie_distribution("muon", "pt")
+specie_distribution("kaon", "eta")
+specie_distribution("pion", "eta")
+specie_distribution("proton", "eta")
+specie_distribution("electon", "eta")
```
Comment (zhangbiao-phy) [inline FirstAnalysis/trackingstudies.py:106]: replace "electon" to "electron"

File: FirstAnalysis/trackingstudies.py [hunk 4]
```diff
@@ -136,42 +78,38 @@ def tracking_resolution(var1, mode, var2, var3):
...
+specie_distribution("muon", "eta")
+specie_distribution("kaon", "phi")
+specie_distribution("pion", "phi")
+specie_distribution("proton", "phi")
+specie_distribution("electon", "phi")
```
Comment (zhangbiao-phy) [inline FirstAnalysis/trackingstudies.py:111]: replace "electon" to "electron"

File: FirstAnalysis/HFmcvalidation.py (hunk 1 above)
Comment (vkucera) [reply FirstAnalysis/HFmcvalidation.py]: It's just needed to remove the spaces around slashes. It has to be a valid path.

File: FirstAnalysis/trackingstudies.py [hunk 5]
```diff
@@ -136,42 +78,38 @@ def tracking_resolution(var1, mode, var2, var3):
...
 
 def vertex_distributions(var):
-    # x, y, z
     fileo2 = TFile("../codeHF/AnalysisResults_O2.root")
     cvertex = TCanvas("cvertex", "vertex ")
     cvertex.SetCanvasSize(1500, 700)
     cvertex.Divide(2, 1)
+    cvertex_res = TCanvas("cvertex", "vertex ")
+    cvertex_res.SetCanvasSize(1500, 700)
+    cvertex_res.Divide(2, 1)
```
Comment (zhangbiao-phy) [inline FirstAnalysis/trackingstudies.py:87]: there are only two plots totally, in line 84, the canvas has already be divided into 2.

**Comment by vkucera** [comment]
@mmazzilli @zhangbiao-phy There seems to be some editing ongoing still. Shall we convert this to a draft until everybody agrees on a final version?

**Comment by vkucera** [comment]
Merging

**Pull request merged**